Makefile

```diff
@@ -747,7 +747,12 @@ endif
 roxvet: $(ROXVET_BIN)
 	@echo "+ $@"
 	@# TODO(ROX-7574): Add options to ignore specific files or paths in roxvet
-	$(SILENT)go vet -vettool "$(ROXVET_BIN)" $(shell go list -e ./... | grep -v 'operator/pkg/clientset')
+	# We can't pipe `go list` output since it exceeds Linux buffer size; using temporary file instead
+	go list -e ./... > go-list-results.txt \
+	grep -v 'operator/pkg/clientset' go-list-results.txt > go-list-filtered-results.txt \
+	go vet -vettool "$(ROXVET_BIN)" $(cat go-list-filtered-results.txt) \
+	rm go-list-results.txt ; \
+	rm go-list-filtered-results.txt ;
```

Comment:
```suggestion
	go list -e ./... | grep -v 'operator/pkg/clientset' > go-list-filtered-results.txt ; \
	go vet -vettool "$(ROXVET_BIN)" $(<go-list-filtered-results.txt) ; \
	rm go-list-filtered-results.txt ;
```

Reply:
We can't use pipe here as the output size goes over the Linux limit

Reply:
Pipes can transfer as much as the output can read, there's no limit.

Reply:
Anyway, #8747 might work better.